### ISSUE TITLE
Redurant call for AddAllSubtypeManagers

### ISSUE THREAD

**Issue by VasiliSker** [buttplugio/buttplug-csharp#610]
In DeviceManager.cs line 269:
```
            if (!_hasAddedSubtypeManagers)
            {
                await AddAllSubtypeManagers();
            }
```
_hasAddedSubtypeManagers doesnt seem to set 'true' elsewhere, suggest use:
```
            if (_managers.Count == 0)
            {
                await AddAllSubtypeManagers();
            }
```

**Comment by qdot** [comment]
Thanks! I'll get this fixed up in 0.4.3, which will hopefully be out in the next day or two. :+1:

**Comment by qdot** [comment]
Fixed in 0.4.3 (but didn't register for some reason.
